mvc/src/test/java/nextstep/mvc/view/JsonViewTest.java

```diff
@@ -0,0 +1,94 @@
+package nextstep.mvc.view;
+
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.times;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import jakarta.servlet.http.HttpServletRequest;
+import jakarta.servlet.http.HttpServletResponse;
+import java.io.PrintWriter;
+import java.util.HashMap;
+import java.util.Map;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+class JsonViewTest {
+
+    @Test
+    @DisplayName("model에 하나의 객체가 담겨있으면 해당 객체를 json으로 마샬링한다")
+    void writeObjectStringWhenHasSingleObject() throws Exception {
+        // given
+        JsonView view = new JsonView();
+
+        Map<String, Object> model = new HashMap<>();
+        Dummy obj = new Dummy("tester", 10);
+        model.put("obj", obj);
+        HttpServletRequest request = mock(HttpServletRequest.class);
+        HttpServletResponse response = mock(HttpServletResponse.class);
+        PrintWriter writer = mock(PrintWriter.class);
+        when(response.getWriter()).thenReturn(writer);
+
+        // when
+        view.render(model, request, response);
+
+        // then
+        String value = writeValueAsString(obj);
+        verify(writer, times(1)).write(value);
+    }
+
+    @Test
+    @DisplayName("하나의 객체가 담겨있지 않으면 Map 자체를 json으로 마샬링한다")
+    void writeMapWhenNotSingleObject() throws Exception{
```

Comment:
꼼꼼한 테스트 👍

사소하지만 공백이 누락되었네요 🥲
 
```suggestion
    @Test
    @DisplayName("하나의 객체가 담겨있지 않으면 Map 자체를 json으로 마샬링한다")
    void writeMapWhenNotSingleObject() throws Exception {
```

Reply:
오 꼼꼼한 리뷰 감사합니다~